qiskit/providers/ibmq/api_v2/rest/version_finder.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Version finder for the IBM Q Api version 2."""
+
+from json import JSONDecodeError
+
+from .base import RestAdapterBase
+
+
+class VersionFinder(RestAdapterBase):
+    """Rest adapter for the version finder."""
+
+    URL_MAP = {
+        'version': '/version'
+    }
+
+    def version_info(self):
+        """Return the version info.
+
+        Returns:
+            dict: a dict of the API versions with a key conveying whether
```

Comment:
It seems a good chance to try to be a bit more accurate about this docstring! A follow-up might be upcoming formalizing a bit the return types in general - in the meantime, can you change to something similar to:

```
dict: a dict with information about the API version, with the following keys:
    * `new_api` (bool): ........
    And the following optional keys:
    * `api` (str??): only for old API ......
    * `api-auth` (str??): new API, auth server ...
    * `api-*` (str?): new API, other servers ...
```